docs/source/chainermn/installation/guide.rst

```diff
@@ -115,14 +114,14 @@ Tested Environments
 We tested ChainerMN on all the following environments.
 
 * OS
-  
+
   * Ubuntu 14.04 LTS 64bit
   * Ubuntu 16.04 LTS 64bit
 
-* Python 2.7.13 3.5.1 3.6.1
+* Python 2.7.13, 3.5.1, 3.6.1
 * MPI
 
-  * openmpi 1.10.7 2.1.2
+  * openmpi 1.10.7, 2.1.2
```

Comment:
Do you mind removing `1.10.7` from tested MPI versions? 
If you think  that change should be in an another PR, it's OK and I am willing to work on it.

Reply:
Ok, I removed it.

Just to make sure, do you still expect OpenMPI 1.10.7 to work with current ChainerMN (you want to remove it just because you don't test that configuration), or it won't work any more?
(If possible, I think it's better to add "Supported Environment (versions)" section in addition to "Tested Environments" section.)